### DISCUSSION TITLE
Chromium security

### DISCUSSION THREAD

**Discussion by ghost** [Alex313031/thorium#772] [Q&A] [answered]
Hi, I see that Throrium is currently based on M126, but doesn't Chromium itself have security vulnerabilities that are patched from M127 onwards?

I'm not familiar with the details, but I've noticed that there have been a lot of reports of security vulnerabilities in Chromium recently.

I am aware that the shutdown of MV2 by Google is met with deep aversion.
However, I don't know if the security vulnerabilities are patched in older Chromium versions like M126. 

Perhaps Google will offer an ESR version at some point, just like Mozilla FireFox. I think that would be a relief for many.

**Comment by RocGod** [comment, marked as answer]
The version will always be 1-2 versions behind the current version, as @Alex313031 is the main person to maintain the code, so he might not have too much time to follow the Chromium release time...
